server/src/service/ReviewsService.js

```diff
@@ -0,0 +1,46 @@
+const reviewsDao = require('../repository/ReviewsDAO');
+const uuid = require('uuid');
+
+async function createNewReview(receivedData) {
```

Comment:
For this function, you are returning too many possible data types.  They include Object, Boolean, or String.  I suggest looking into throwing errors.